### PR TITLE
Mdd mcc fixes

### PR DESCRIPTION
**Pull request by mddifilippo89**
Massive PR 158 files

Mostly build flag issues with the modules and blazor.
Added snippets and content to MCC topic for React and WC etc.

### CODE REVIEW

File: doc/en/components/data-chart-navigation.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ In the $PlatformShort$ data chart component, navigation is disabled by default.
 
 In order to use navigation in the data chart, you need to import and register the interactivity module. This can be done by using the following code:
 
-<!-- Blazor -->
-* DataChartInteractivityModule
-<!-- end: Blazor -->
+```razor
+DataChartInteractivityModule
```
Comment (HUSSAR-mtrela) [inline doc/en/components/data-chart-navigation.md:28]: append `.Register(IgniteUIBlazor);`

File: doc/en/components/category-chart.md [hunk 2]
```diff
@@ -53,14 +53,12 @@ npm install --save {PackageCharts}
...
 
-<!-- Blazor -->
-The `CategoryChart` requires the following modules<!-- Angular, React, WebComponents -->.<!-- end: Angular, React, WebComponents --><!-- Blazor --> to be registered in the AddIgniteUIBlazor extension method in your application entry point:
+The `CategoryChart` requires the following modules.
 
-* CategoryChartModule
-
-<!-- end: Blazor -->
+```razor
+CategoryChartModule
```
Comment (HUSSAR-mtrela) [inline doc/en/components/category-chart.md:59]: append `.Register(IgniteUIBlazor);`

File: doc/en/components/data-chart-series-highlighting.md [hunk 3]
```diff
@@ -27,10 +27,10 @@ The $PlatformShort$ data chart component allows you to highlight the plotted ser
 
 In order to enable highlighting in the $PlatformShort$ data chart component, you need to import and register the interactivity module. This can be done using the following code:
 
-<!-- Blazor -->
-* DataChartInteractivityModule
-* DataChartAnnotationModule
-<!-- end: Blazor -->
+```razor
+DataChartInteractivityModule
+DataChartAnnotationModule
```
Comment (HUSSAR-mtrela) [inline doc/en/components/data-chart-series-highlighting.md:32]: append `.Register(IgniteUIBlazor);`

File: doc/en/components/data-chart-series-tooltips.md [hunk 4]
```diff
@@ -24,9 +24,9 @@ The $PlatformShort$ data chart component has support for showing tooltips along
 
 In order to use tooltips or tooltip layers in the $PlatformShort$ data chart component, you need to import and register the interactivity module. This can be done by using the following code:
 
-<!-- Blazor -->
-* DataChartInteractivityModule
-<!-- end: Blazor -->
+```razor
+DataChartInteractivityModule
```
Comment (HUSSAR-mtrela) [inline doc/en/components/data-chart-series-tooltips.md:28]: append `.Register(IgniteUIBlazor);`

File: doc/en/components/data-chart-type-category-bar-series.md [hunk 5]
```diff
@@ -44,13 +44,14 @@ public dataSource: any[] = SampleCategoryData.create();
 
 ## Required Modules
 
-Creation of the `BarSeries` requires the following modules<!-- Angular, React, WebComponents -->.<!-- end: Angular, React, WebComponents --><!-- Blazor --> to be registered in your application entry point:
+Creation of the `BarSeries` requires the following modules.
```
Comment (HUSSAR-mtrela) [inline doc/en/components/data-chart-type-category-bar-series.md:47]: replace "." with ":"

File: doc/kr/components/spreadsheet_overview.md [hunk 6]
```diff
@@ -32,11 +32,12 @@ When installing the spreadsheet package, the core and excel package must also be
...
 
-The `Spreadsheet` requires the following modules<!-- Angular, React, WebComponents -->.<!-- end: Angular, React, WebComponents --><!-- Blazor --> to be registered in your application entry point:
+The `Spreadsheet` requires the following modules.
 
-* ExcelModule
+```razor
+ExcelModule
 * SpreadsheetModule
-<!-- end: Blazor -->
+```
```
Comment (HUSSAR-mtrela) [inline doc/kr/components/spreadsheet_overview.md:40]: remove "*" and append `.Register(IgniteUIBlazor);`

File: doc/en/components/multi-column-combobox.md [hunk 7]
```diff
@@ -8,25 +8,24 @@ mentionedTypes: []
...
 The Multi-Column Combo Box automatically generates columns for properties on the data object. This component is unique in that it's a combo box that visualizes large amounts of data similar to a data grid embedded in the dropdown. 
 
-Data binding can be achieved using an array of complex objects via the column's `DataSource` property.
+## Usage
 
-The `TextField` property determines which value is shown when users make a selection. 
+The following properties are the most commonly used to configure the component:
 
-The `ValueField` property determines the bound value of the underlying data item selected. This is necessary if your list of objects have several properties.
+- `DataSource` - Data binding can be achieved using an array of complex objects via the column's `DataSource` property.
```
Comment (HUSSAR-mtrela) [inline doc/en/components/multi-column-combobox.md:15]: change description to:
allows binding data in form of an array of complex objects that will be displayed in drop down menu

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]
these code snippets are still incomplete because they are missing:
`CategoryChartModule.Register(IgniteUIBlazor);`
OR:
`builder.Services.AddIgniteUIBlazor(typeof(CategoryChartModule));`

File: doc/en/components/data-chart-navigation.md (hunk 1 above)
Comment (mddifilippo89) [reply doc/en/components/data-chart-navigation.md]: done

File: doc/en/components/category-chart.md (hunk 2 above)
Comment (mddifilippo89) [reply doc/en/components/category-chart.md]: done

File: doc/en/components/data-chart-series-highlighting.md (hunk 3 above)
Comment (mddifilippo89) [reply doc/en/components/data-chart-series-highlighting.md]: done

File: doc/en/components/data-chart-series-tooltips.md (hunk 4 above)
Comment (mddifilippo89) [reply doc/en/components/data-chart-series-tooltips.md]: done

File: doc/en/components/data-chart-type-category-bar-series.md (hunk 5 above)
Comment (mddifilippo89) [reply doc/en/components/data-chart-type-category-bar-series.md]: done

File: doc/kr/components/spreadsheet_overview.md (hunk 6 above)
Comment (mddifilippo89) [reply doc/kr/components/spreadsheet_overview.md:40]: done

File: doc/en/components/multi-column-combobox.md (hunk 7 above)
Comment (mddifilippo89) [reply doc/en/components/multi-column-combobox.md]: done

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**